cluster/common.sh

```diff
@@ -942,9 +942,9 @@ EOF
 DISABLE_NETWORK_SERVICE_SUPPORT: $(yaml-quote ${DISABLE_NETWORK_SERVICE_SUPPORT})
 EOF
     fi
-    if [ -n "${DISABLE_ADMISSION_PLUGINS:-}" ]; then
+    if [ -n "${NETWORK_PROVIDER:-}" ]; then
       cat >>$file <<EOF
-DISABLE_ADMISSION_PLUGINS: $(yaml-quote ${DISABLE_ADMISSION_PLUGINS})
+NETWORK_PROVIDER: $(yaml-quote ${NETWORK_PROVIDER})
```

Comment:
This already included in the same function:  line#792
```
NETWORK_PROVIDER: $(yaml-quote ${NETWORK_PROVIDER:-})
```

Reply:
Thank you. good to know. will remove this line.

Reply:
Removed duplicate codes.